src/app/lib/config/services/afaanoromoo.js

```diff
@@ -129,39 +129,31 @@ const service = {
   footer: {
     externalLink: {
       href: 'https://www.bbc.co.uk/help/web/links/',
-      text: 'Read about our approach to external linking.',
+      text: 'Itti dhiyaachuu keenya gara geessituu.',
     },
     links: [
       {
-        href: 'https://www.bbc.com/news/help-41670342',
-        text: 'Why you can trust the BBC',
+        href: 'https://www.bbc.com/afaanoromoo/institutional-49281861',
+        text: 'BBC News maaliif amanuu dandeessa',
       },
       {
-        href: 'https://www.bbc.com/terms',
-        text: 'Terms of Use',
+        href: 'https://www.bbc.co.uk/usingthebbc/terms/',
+        text: 'Haala itti fayyadamaa',
       },
       {
-        href: 'https://www.bbc.co.uk/aboutthebbc/',
```

Comment:
Are we meaning to lose the `aboutthebbc` link from the footer? It's currently on the production site https://www.bbc.com/afaanoromoo

Reply:
Not sure - these were the only translations provided

I asked Jane over on the issue: https://github.com/bbc/simorgh/issues/3034